src/app/features/test-records/create/views/create-test-record/create-test-record.component.html

```diff
@@ -1,25 +1,43 @@
 <ng-container *ngIf="testResult$ | async as testResult">
-  <ng-container *ngIf="!isAbandon; else abandoned">
-    <app-base-test-record
-      [isEditing]="true"
-      [testResult]="testResult"
-      [expandSections]="true"
-      (newTestResult)="handleNewTestResult($event)"
-    ></app-base-test-record>
+  <ng-container [ngSwitch]="testMode">
```

Comment:
You can combine this and the above `<ng-container>`
```suggestion
<ng-container *ngIf="testResult$ | async as testResult" [ngSwitch]="testMode">
```